rangeserver/src/epoch_supplier.rs

```diff
@@ -1,10 +1,5 @@
-use thiserror::Error;
-
-#[derive(Debug, Error)]
-pub enum Error {
-    #[error("Unknown")]
-    Unknown,
-}
+pub mod reader;
+use epoch_publisher::error::Error;
 
 pub trait EpochSupplier: Send + Sync + 'static {
```

Comment:
Would this be a good candidate to be a `async_trait`? 
https://docs.rs/async-trait/latest/async_trait/

Reply:
Nice! Done, thanks!